aperturedb/Utils.py

```diff
@@ -415,3 +415,38 @@ def get_indexed_props(self, class_name, type="entities"):
             indexed_props = []
 
         return indexed_props
+
+    def count_descriptors_in_set(self, set_name):
+
+        q = [{
+            "FindDescriptorSet": {
+                "_ref": 1,
+                "with_name": set_name,
+
+            }
+        }, {
+            "FindDescriptor": {
+                "set": set_name,
+                "is_connected_to": {
+                    "ref": 1,
+                    "connection_class": "_DescriptorSetToDescriptor"
```

Comment:
is "ref": 1 not sufficient ?

Reply:
not really, because you can have connection other than _DescriptorSetToDescriptor. We just want to make sure we only count those.